### ISSUE TITLE
Emphasize that the schema for all conventional datastores must be the same

### ISSUE THREAD

**Issue by rgwilton** [netmod-wg/datastore-dt#2]
RobW raised on 11/9/17:

Old:

```
4.5.  Conventional Configuration Datastores

   The conventional configuration datastores are a set of configuration
   datastores that share a common schema, allowing data to be copied
   between them.  The term is meant as a generic umbrella description of
   these datastores.  The set of datastores include:
```

New:

```
4.5.  Conventional Configuration Datastores

   The conventional configuration datastores are a set of configuration
   datastores that all share exactly the same schema, allowing data to be copied
   between them.  The term is meant as a generic umbrella description of
   these datastores.  The set of datastores include:
```

**Comment by rgwilton** [comment]
Latest proposed text after discussion from Kent, Martin, and RobW:

```
   The conventional configuration datastores are a set of configuration
   datastores that share exactly the same schema, allowing data to be copied
   between them.  The term is meant as a generic umbrella description of
   these datastores.  The set of datastores include:
```

**Comment by mbj4668** [comment]
Fixed in eca23d9172c62ec93939966b0989c91b30a85651.
